.gitignore

```diff
@@ -1,5 +1,6 @@
 src/info.py
-
+apps/beat.py
```

Comment:
this should not be in the ignore

Reply:
I didn't want to make a mess in this file.